README.md

```diff
@@ -126,8 +126,8 @@ wallet.verify(message, signature); // true
 ```javascript
 const { XpringClient } = require("xpring-js");
 
-const remoteURL = "grpc.xpring.tech:80";
-const xpringClient = XpringClient.xpringClientWithEndpoint(remoteURL);
+const remoteURL = "3.14.64.116:50051"; // TestNet URL, use 18.222.225.81:50051 for MainNet
+const xpringClient = XpringClient.xpringClientWithEndpoint(remoteURL, true);
```

Comment:
This is the only xpringClient that is constructed with `xpringClientWithEndpoint`... is that correct or should this also use `new` ?

Reply:
Thank you!